apps/CEventService.API/obj/CEventService.API.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
remove this obj folder from the PR

Reply:
Those files do not correspond to my files, it is as if I have pulled files from another branch.